Sanity/mariadb-odbc/main.fmf

```diff
@@ -0,0 +1,23 @@
+summary: Basic smoke-test for accessing MariaDb via unixODBC
+contact:
+- Filip Janus <fjanus@redhat.com>
+component:
+- python3-pyodbc
+- unixODBC
+- mariadb-connector-odbc
+- mariadb
+test: ./runtest.sh
+framework: beakerlib
+recommend:
+- python3-pyodbc
```

Comment:
This package duplication is intended?

Reply:
What duplication do you mean?

Reply:
component means for which component is the test suited to be ran

recommend means the same as requires in rpm spec -> which packages are required to be installed for the test to run properly